packages/release/media-cdrom.mount

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=CDROM mount(/media/cdrom)
```

Comment:
I'm nothing but a causal bystander who enjoys the thought of this PR, but IMO this would probably look better (if someone were watching the boot logs) with a little extra punctuation, ala `Description=CD-ROM mount (/media/cdrom)` :innocent: